Source/Core/TerrainProvider.js

```diff
@@ -227,5 +227,15 @@ define([
      */
     TerrainProvider.prototype.getTileDataAvailable = DeveloperError.throwInstantiationError;
 
+        /**
+     * Makes sure we load availability data for a tile
+     *
+     * @param {Number} x The X coordinate of the tile for which to request geometry.
+     * @param {Number} y The Y coordinate of the tile for which to request geometry.
+     * @param {Number} level The level of the tile for which to request geometry.
+     * @returns {undefined|Promise} Undefined if nothing need to be loaded or a Promise that resolves when all required tiles are loaded
+     */
+    TerrainProvider.prototype.loadTileDataAvailability = DeveloperError.throwInstantiationError;
```

Comment:
This is a breaking change because existing terrain providers people have written will need to be updated.  While that's unlikely to affected most (all?) of our users, we should list it as such in CHANGES

Reply:
Basically, if they implement availability, they need to implement loadTileDataAvailability, right?